src/UserManager.ts

```diff
@@ -513,13 +516,13 @@ export class UserManager {
                 url: signoutRequest.url,
                 state: signoutRequest.state?.id,
             });
-        }
-        catch (err) {
+        } catch (err) {
             logger.debug("error after preparing navigator, closing navigator window");
             handle.close();
             throw err;
         }
     }
+
```

Comment:
please avoid changing code style,

Reply:
I am sorry, this is something that my IDE does if not otherwise configured. Perhaps you should consider enforcing this specific rule as well via the .editorconfig file if possible.